doc/integrator/database.rst

```diff
@@ -3,6 +3,16 @@
 Database
 ========
 
+Schema
+------
+
+The database table are separate in different schemas:
+- The 'main' schema, can be configured with the ``PGSCHEMA`` environment variable, contains the data that can be modified only by the administrator interface.
+- The 'static' schema, can be configured with the ``PGSCHEMA_STATIC`` environment variable, contains the data that can be modified by the end used through the web application.
```

Comment:
I don't understand what is meant, which data? What data can be modified through the web application and not via administrator interface? Do you mean the editing functionality?